MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,423 @@
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Selected segment border thickness customization
+You can customize the selected segment border thickness, by using `StrokeThickness` property of `SelectionIndicatorSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.SelectionIndicatorSettings>
+            <buttons:SelectionIndicatorSettings 
+                StrokeThickness="1"/>
+        </buttons:SfSegmentedControl.SelectionIndicatorSettings>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            StrokeThickness = 1,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Selection changed event
+The `SelectionChanged` event is triggered, once the selection process has been completed for the selected item in the Segmented Control. The `SelectionChangedEventArgs` has the following values, which provides information for `SelectionChanged` event:
+
+* OldIndex
+* NewIndex
+* OldValue
+* NewValue
+
+{% tabs %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionChanged += OnSegmentedControlSelectionChanged;
+        this.Content = segmentedControl;
+    }
+
+    private void OnSegmentedControlSelectionChanged(object sender, Syncfusion.Maui.Buttons.SelectionChangedEventArgs e)
+    {
+        var newValue = e.NewValue;
+        var oldValue = e.OldValue;
+        var newIndex = e.NewIndex;
+        var oldIndex = e.OldIndex;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Keyboard navigation
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves to the next item on the segmented control's right side. It does not make any selection and only navigates to the next item. If the focus border is on the last item, pressing RightArrow does nothing.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves to the previous item on the segmented control's left side. It does not make any selection and only navigates to the previous item. If the focus border is on the first item, pressing LeftArrow does nothing.<br/><br/></td></tr>
```

Comment:
Add Enter, which used to select the segment

Reply:
Added Enter